cluster/common.sh

```diff
@@ -995,7 +995,12 @@ function generate-certs {
     mv "kubelet.pem" "pki/issued/kubelet.crt"
     rm -f "kubelet.csr"
 
-    ./easyrsa build-client-full kubecfg nopass
+    # Make a superuser client cert with subject "O=system:masters, CN=kubecfg"
+    ./easyrsa --dn-mode=org \
+      --req-cn=kubecfg --req-org=system:masters \
+      --req-c= --req-st= --req-city= --req-email= --req-ou= \
+      build-client-full kubecfg nopass
```

Comment:
I think that's a reasonable answer for the old user-facing client certs. If you want to migrate to RBAC-only, but want your cert to keep having admin permissions, you should create a binding before making it lose power.

On GCE, the client cert is still probably the best authn method, so it makes sense to keep supporting it w/ the new model (add the system:masters group to new certs). I'm going to punt on it right now for GKE, but I'd like it if we no longer minted new per-cluster admin certs.